starter-code/public/admin.html

```diff
@@ -50,9 +61,9 @@ <h2>Author Stats</h2>
     <script src="/scripts/article.js"></script>
     <script src="/scripts/articleView.js"></script>
     <script>
-      // TODO: After we fetch and load all the articles, what view function do we want to run next?
+      // TODO - COMPLETED: After we fetch and load all the articles, what view function do we want to run next?
       // Let's pass in a callback function, to determine what happens AFTER all articles are loaded.
-      Article.fetchAll();
+      Article.fetchAll(articleView.initIndexPage);
```

Comment:
should be `Article.fetchAll(articleView.initAdminPage);`
since this is `admin.html` you want to call `articleView.initAdminPage` so that the author stats will load